scripts/restore

```diff
@@ -48,30 +48,30 @@ fi
 #=================================================
 # RESTORE THE CRON FILE
 #=================================================
-ynh_script_progression --message="Restoring cron job..." --weight=1
+ynh_script_progression "Restoring cron job..."
 
-ynh_restore_file --origin_path="/etc/cron.d/$app"
+ynh_restore "/etc/cron.d/$app"
 
 #=================================================
 # RESTORE LOGS
 #=================================================
 
-ynh_restore_file --origin_path="/var/log/$app"
+ynh_restore "/var/log/$app"
 
 #=================================================
 # RESTORE THE LOGROTATE CONFIGURATION
 #=================================================
-ynh_script_progression --message="Restoring the logrotate configuration..." --weight=1
+ynh_script_progression "Restoring the logrotate configuration..."
 
-ynh_restore_file --origin_path="/etc/logrotate.d/$app"
+ynh_restore "/etc/logrotate.d/$app"
 
 #=================================================
 # RESTORE THE DATA DIRECTORY
 #=================================================
-ynh_script_progression --message="Restoring data directory..." --weight=2
+ynh_script_progression "Restoring data directory..."
 
-# Use --not_mandatory for the data directory, because if the backup has been made with BACKUP_CORE_ONLY, there's no data into the backup.
-ynh_restore_file --origin_path="$data_dir" --not_mandatory
+# Use || true for the data directory, because if the backup has been made with BACKUP_CORE_ONLY, there's no data into the backup.
```

Comment:
```suggestion
```

(no need to have || true if i remember correctly, everything is handled implicitly by the helpers)